examples/ex42.cpp

```diff
@@ -1376,6 +1302,15 @@ ElasticityOperator::ElasticityOperator(ParMesh &mesh, const int order)
    cstate_local_.SetSize(h1_prolongation_->Height());
 
    gradient_ = new ElasticityGradientOperator(*this);
+
+   BG2_[0] = make_tensor<2,2>([&](int i, int j) { return maps_->B[i+2*j]; });
+   BG2_[1] = make_tensor<2,2>([&](int i, int j) { return maps_->G[i+2*j]; });
+
+   BG3_[0] = make_tensor<3,3>([&](int i, int j) { return maps_->B[i+3*j]; });
+   BG3_[1] = make_tensor<3,3>([&](int i, int j) { return maps_->G[i+3*j]; });
+
+   BG4_[0] = make_tensor<4,4>([&](int i, int j) { return maps_->B[i+4*j]; });
+   BG4_[1] = make_tensor<4,4>([&](int i, int j) { return maps_->G[i+4*j]; });
```

Comment:
Why are those kernel dependent in the dispatch but not here? Why is this needed?